ssc/cmod_merchantplant.cpp

```diff
@@ -2992,11 +2996,14 @@ class cm_merchantplant : public compute_module
 		save_cf( CF_property_tax_assessed_value, nyears, "cf_property_tax_assessed_value" );
 		save_cf( CF_property_tax_expense, nyears, "cf_property_tax_expense" );
 		save_cf( CF_insurance_expense, nyears, "cf_insurance_expense" );
-		save_cf(CF_battery_replacement_cost, nyears, "cf_battery_replacement_cost");
-		save_cf(CF_battery_replacement_cost_schedule, nyears, "cf_battery_replacement_cost_schedule");
-		save_cf(CF_fuelcell_replacement_cost, nyears, "cf_fuelcell_replacement_cost");
-		save_cf(CF_fuelcell_replacement_cost_schedule, nyears, "cf_fuelcell_replacement_cost_schedule");
-		save_cf( CF_operating_expenses, nyears, "cf_operating_expenses" );
+        if ((as_integer("en_batt") == 1) && (as_integer("batt_replacement_option") > 0)) {
```

Comment:
Mind removing the second half of this if statement ("batt_replacement_option")? Some of the work I'm doing right now compares cases using "no replacement" to various replacement strategies, so having zeroes come out of SSC is useful to that workflow.